### DISCUSSION TITLE
Is this project open source

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [caido/caido#1620] [Q&A]
Hi all,

I've been looking through the code but couldn't find any rust files. Is this project open-source or closed-source? if open, where can one find the code?

Thank you,
Mikey Ross

**Comment by Sytten** [comment]
Hi!

Caido is not currently open source, we build in public and oss some components where it makes sense. We also offer an API for open source plugin development and employ standard open formats whenever possible.

Emile

**Comment by timdelange** [comment]
Pity, most security workers feel safer using open source with many eyes  on the  code.
